app/models/post.rb

```diff
@@ -0,0 +1,14 @@
+class Post < ApplicationRecord
+  has_many :comments
+  has_many :likes
+  belongs_to :user, counter_cache: true
```

Comment:
- [ ] You are right @BenM4k, in my comment, I did not ask you to change anything in the like and comment model, just to update the post as you did and it's okay, this is the reason why i added an image of your post-migration and not the like and comment because I saw you did not use author to reference the user, hope this calm you dear @BenM4k

Reply:
If you see my commits, I changed the 3 files as you requested to change likes and comments too, but the posts were deleted, it's okay. I'm getting an extra review, but it's not the world's end.

Reply:
deleted before I submit the review summary, means you started working on the changes before I submitted the review what you should not do as the review was still under process @BenM4k , anyways, it's okay